### DISCUSSION TITLE
[QUESTION] gandlf: command not found

### DISCUSSION THREAD

**Discussion by ujjwalbaid0408** [mlcommons/GaNDLF#928] [Q&A] [answered]
### Environment information
 gandlf: command not found

GaNDLF version, OS, and any other relevant information.

GaNDLF version: 0.0.20
OS: Windows

### How did you install GaNDLF
Please provide all steps followed during installation.
pip install gandlf

### Dataset description
Describe the data (radiology/histology/so on, dimensions, etc.) with as much detail as possible.
Radiology data with dimension 512*512*1 for classification task

### Describe your question/problem
A clear and concise description of what issue you are facing.

When the gandlf run command is executed, error message as  'gandlf: command not found' is received.
However if gandlf_run (underscore between gandlf and run) is used then code runs as expected.

**Comment by sarthakpati** [comment, marked as answer]
Thanks for the question!

The `gandlf ABC` routines are only available from the version `0.1.0` and later (which as of today is the current master branch and not tagged). You should be able to run the previous commands (i.e., `gandlf_run`).
